tests/spread/plugins/craft-parts/build-and-run-hello/matter-hello/snap/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: matter-hello
+version: "1.0"
+summary: simple matter application
+description: build an example lighting application using the Matter SDK and say "hello world"
+base: core22
+confinement: strict
+
+apps:
+  matter-hello:
+    command: bin/lighting-app
+
+parts:
+  lighting-hello:
+    plugin: matter
+    matter-sdk-version: "master"
```

Comment:
This will not make up a stable reference because any breaking upstream change will result in failure in the tests. The goal of the tests are to ensure changes to snapcraft don't cause regression in plugins.

Reply:
RIght, I am aware that it is not using a stable upstream release. However, currently, the latest stable version, v1.2.0.1, is broken due to the Python dependency issue described here: https://github.com/project-chip/connectedhomeip/issues/30981

Reply:
And sooner or later, `master` will break for some other reason. Snapcraft tests cannot rely on a development and moving target.

At least this should be changed to a commit known to be stable for this purpose and changes to a release tag as soon as available.

Reply:
Updated to use a specific commit with a fix.